### PR TITLE
Add comments to status updates

### PR DESCRIPTION
**Pull request by F3PiX**
Taking the onelegant but pragmatic route: this is in fact #528, but with a brand new PR.

### CODE REVIEW

File: app/controllers/status_update_comments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  def create
+    comment = Comment.new(comment_params)
+    status_update = comment.commentable
+
+    if comment.text.present?
+      comment.save
+    else
+      flash[:alert] = "O no! We can't save an empty comment. Please try again?"
+    end
+     redirect_to status_update_path(status_update)
```
Comment (carpodaster) [inline app/controllers/status_update_comments_controller.rb]: Indentation

File: app/controllers/status_update_comments_controller.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  end
+
+  private
+
+  def comment_params
+    params.require(:comment).permit(:commentable_id, :commentable_type, :text).merge(user_id: current_user.id)
+  end
+
+  end
```
Comment (carpodaster) [inline app/controllers/status_update_comments_controller.rb]: Indentation

File: app/models/activity.rb [hunk 3]
```diff
@@ -2,6 +2,8 @@ class Activity < ActiveRecord::Base
   KINDS = %w(feed_entry mailing status_update)
 
   belongs_to :team
+  has_many :comments, -> { ordered }, as: :commentable, dependent: :destroy
+
```
Comment (carpodaster) [inline app/models/activity.rb]: Superfluous new line

**Comment by carpodaster** [comment]
LET'S DO THIS! 😄 

Thanks a lot Maud!

**Comment by F3PiX** [comment]
🎈 Thnx Carsten and Michael, for your help, I learned a lot.

**Pull request merged**